robot.py

```diff
@@ -58,10 +59,15 @@ def createObjects(self) -> None:
         self.chassis_4_drive = ctre.WPI_TalonFX(7)
         self.chassis_4_steer = ctre.WPI_TalonFX(8)
 
-        self.joystick = wpilib.Joystick(0)
-        self.recorded_joystick_state = (0.0, 0.0, 0.0)
+        self.recorded_drive_state = (0.0, 0.0, 0.0)
         self.recorded_is_local_driving = False
-        self.codriver = wpilib.XboxController(1)
+        self.xbox_controller = wpilib.XboxController(0)
+        # (time to reach 63% of final value, rate)
+        self.joystick_x_filter = LinearFilter.singlePoleIIR(0.05, 0.02)
+        self.joystick_y_filter = LinearFilter.singlePoleIIR(0.05, 0.02)
+        self.joystick_z_filter = LinearFilter.singlePoleIIR(0.3, 0.02)
+        # joystick used for test mode
+        self.joystick = wpilib.Joystick(1)
```

Comment:
Maybe keep the slots for the HIDs the same, so we don't have to worry about swapping them when merging or we decide to revert.